frontend/src/components/CustomTableForm.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <div>
+  <div class="form-class-text">
+    <h4 style="padding-left: 30px"> You can start creating your custom table here by checking the UL campaigns you're following: </h4>
+  <div class="form-class" style="text-align: center">
+    <span class="input-class"  v-for="campaign in existingCampaigns" :key="campaign"><label for="campaign.name">{{campaign.name}}</label>
+    <input type="checkbox" v-on:change="generateURL" v-bind:id="campaign.name" v-bind:value="campaign.name" v-model="campaigns"></span>
+  </div>
+  
+    <h4 style="padding-left: 30px"><label for="datasetnames">Please enter the dataset names you want to follow in your table separated by a | below with no spaces: </label></h4>
+    <input style="text-align: center; color: grey; width: 60%;background-color: white;border-style: solid;" v-model="datasetnames" v-on:input="generateURL" id="datasetnames" placeholder="TTToHadronic_|ST_t-channel_top_4f_">
+    
+    <p style="padding-left: 30px">Your custom table is here: <a v-bind:href=url> {{url}}</a> </p>    
+  </div>
+  </div>
+</template>
+
+<script>
+import axios from 'axios'
+
+export default {
+  name: 'CustomTableForm',
+  props:['existingCampaigns'],
+  data () {
+    return {
+    campaigns: [],
+    datasetnames: '',
+    url: 'https://cms-pdmv.cern.ch/grasp/'
+    }
+  },
+  created () {
+    this.fetchObjectsInfo();
+  },
+  methods: {
+      generateURL: function() {
+          const campaign_holder = this.campaigns
+          let url = 'https://cms-pdmv.cern.ch/grasp/existing?name=' + campaign_holder.join(',') + '&dataset=' + this.datasetnames 
```

Comment:
1. No need to hardcode url here as well, just do `let url = 'existing?name=' + this.campaigns.join(',') + '&dataset=' + this.datasetnames` OR if you really want the full url, you can do `location.origin + location.pathname`, because now it always points to production GrASP - I can't even properly test it in my instance without changing the code.
1. It would look nicer to put `'&dataset='...` inside an if,so urls without dataset would be cleaner.
1. ALSO, it would make sense to hide "generated url" if no campaigns are selected, otherwise the link is nonsensical `?name=&dataset=`